test/built-ins/TypedArray/prototype/entries/resizable-buffer.js

```diff
@@ -0,0 +1,180 @@
+// Copyright 2023 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-%typedarray%.prototype.values
+description: >
+  TypedArray.p.values behaves correctly when receiver is backed by resizable
+  buffer
+includes: [compareArray.js, resizableArrayBufferUtils.js]
+features: [resizable-arraybuffer]
+---*/
+
+function TypedArrayEntriesHelper(ta) {
+  return ta.entries();
+}
```

Comment:
Also removed unnecessary abstraction as commented in https://github.com/tc39/test262/pull/4072/files#r1594455691